### ISSUE TITLE
Cannot train my own dataset

### ISSUE THREAD

**Issue by Lee-JeongMin** [thtrieu/darkflow#1025]
$ ./flow --model ./cfg/my-yolo.cfg --labels ./labels.txt --dataset ./data/corn_img/ --annotation ./data/annotation_corn --batch 32 --epoche 1000 --keep 50 --lr 1e-04 --gpu 0.8 --gpuName "/gpu:2" --gpuName "gpu:3"

Parsing ./cfg/my-yolo.cfg
Loading None ...
Finished in 0.00010848045349121094s

Building net ...
Source | Train? | Layer description                | Output size
-------+--------+----------------------------------+---------------
WARNING:tensorflow:From /usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
       |        | input                            | (?, 608, 608, 3)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 608, 608, 32)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 304, 304, 32)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 304, 304, 64)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 152, 152, 64)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 152, 152, 128)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 152, 152, 64)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 152, 152, 128)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 76, 76, 128)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 76, 76, 256)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 76, 76, 128)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 76, 76, 256)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 38, 38, 256)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 38, 38, 512)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 38, 38, 256)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 38, 38, 512)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 38, 38, 256)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 38, 38, 512)
 Load  |  Yep!  | maxp 2x2p0_2                     | (?, 19, 19, 512)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 19, 19, 1024)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 19, 19, 512)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 19, 19, 1024)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 19, 19, 512)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 19, 19, 1024)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 19, 19, 1024)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 19, 19, 1024)
 Load  |  Yep!  | concat [16]                      | (?, 38, 38, 512)
 Init  |  Yep!  | conv 1x1p0_1  +bnorm  leaky      | (?, 38, 38, 64)
 Load  |  Yep!  | local flatten 2x2                | (?, 19, 19, 256)
 Load  |  Yep!  | concat [27, 24]                  | (?, 19, 19, 1280)
 Init  |  Yep!  | conv 3x3p1_1  +bnorm  leaky      | (?, 19, 19, 1024)
 Init  |  Yep!  | conv 1x1p0_1    linear           | (?, 19, 19, 30)
-------+--------+----------------------------------+---------------
GPU mode with 0.8 usage
2019-04-16 12:51:27.968400: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-04-16 12:51:27.979562: E tensorflow/stream_executor/cuda/cuda_driver.cc:300] failed call to cuInit: CUDA_ERROR_UNKNOWN: unknown error
2019-04-16 12:51:27.979625: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:161] retrieving CUDA diagnostic information for host: HOSTNAME
2019-04-16 12:51:27.979647: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:168] hostname: HOSTNAME
2019-04-16 12:51:27.979718: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:192] libcuda reported version is: 410.48.0
2019-04-16 12:51:27.979766: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:196] kernel reported version is: 390.116.0
2019-04-16 12:51:27.979787: E tensorflow/stream_executor/cuda/cuda_diagnostics.cc:306] kernel version 390.116.0 does not match DSO version 410.48.0 -- cannot find working devices in this configuration
2019-04-16 12:51:28.000299: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3598230000 Hz
2019-04-16 12:51:28.001093: I tensorflow/compiler/xla/service/service.cc:150] XLA service 0x57ae4e0 executing computations on platform Host. Devices:
2019-04-16 12:51:28.001137: I tensorflow/compiler/xla/service/service.cc:158]   StreamExecutor device (0): <undefined>, <undefined>
Finished in 4.6199986934661865s

Forwarding 8 inputs ...
Total time = 2.9939000606536865s / 8 inps = 2.6720998824033173 ips
Post processing 8 inputs ...
Total time = 0.806816577911377s / 8 inps = 9.91551266919895 ips

------------------------------------------------------------------------------------------------------
When I try "train", it ends here.
How can I change?

**Comment by skofot** [comment]
I believe you missed the `--train` command. I'd also strongly recommend that you use pre-trained weights for you respective config,  weights found [here](https://pjreddie.com/darknet/yolo/) and loaded with `--load file-path`. In my experience, using `--trainer adam` as well gives a far better trained model in the end.

**Comment by Lee-JeongMin** [comment]
@skofot Thank you :-) I was sucess to train.
